apps/pie-docs/src/__tests__/_utilities/navigation.test.js

```diff
@@ -5,6 +5,6 @@ describe('PIE Docs - Navigation', () => {
     it('should compare the actual navigation routes to the JSON snapshot.', () => {
         const routes = getNavigationRoutes();
 
-        expect(routes, 'If you believe this route is expected, please run yarn test:generate-routes to regenerate the expected routes snapshot.').toEqual(expectedRoutes);
```

Comment:
What was wrong with this? The failure message looks helpful to me

Reply:
After adding the Vitest eslint rules, it was saying to remove this. It looks like it's not even a valid function argument!